sapi/cli/tests/cli_header_handler.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+CLI header(), header_remove() and headers_list()
+--SKIPIF--
+<?php include "skipif.inc" ?>
+--INI--
+expose_php=On
```

Comment:
I would leave the `X-Powered-By` header test separately and remove it by default in the others.